eng/tox/tox_helper_tasks.py

```diff
@@ -50,10 +50,15 @@ def setup(*args, **kwargs):
     os.chdir(current_dir)
     _, kwargs = global_vars["__setup_calls__"][0]
 
-    return kwargs["name"], kwargs["version"]
+    package_name = kwargs["name"]
+    # default namespace for the package
+    name_space = package_name.replace('-', '.')
+    if "packages" in kwargs.keys():
+        packages = kwargs["packages"]
```

Comment:
@lmazuel : This change here is to identify the namespace of a given package from kwargs["packages"]. Do you see any issue in this approach?

Reply:
should be fine